contributor_docs/access.md

```diff
@@ -0,0 +1,36 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means considering the vectors of diversity (e.g. gender, social, economic, race, ethnicity, language, ability, etc.) that can impact access/participation; and taking action to acknowledge, dismantle, and prevent barriers. We prioritize the needs of historically marginalized groups over the continued comfort of more privileged groups with p5.js.
+
+This is part of an ongoing conversation about access and inclusion within p5.js. Our intention to hold these as core values from which p5.js is built is laid out in our [Community Statement](../CODE_OF_CONDUCT.md), which was written at the [2015 Contributors Conference](https://p5js.org/community/contributors-conference-2015.html).
+
+**Please consider this a starting point.** We want to invite more conversations about what access means and how we can prioritize it.
+
+## Kinds of access
+
+Increasing access is not a focus on expanding the raw number of people in the p5.js community. It is a focus on making p5.js available to and approachable for people who are excluded from the p5.js community (intentionally or not) and from similar tools and communities.
+
+Access here means making p5.js better for:
+
+- People who speak languages other than English
+- Black people, Indigenous peoples, and People of Color
+- People who are lesbian, gay, bisexual, trans, or queer
+- People with marginalized genders
+- People with disabilities or illness
+- People with little or no prior experience in open source and creative coding 
```

Comment:
my one nit-picky question here is how do we determine "little prior experience"? would it include anyone that is a student? would it include someone that has mastered the basics of coding? i'm trying to get specific here because i see it would be easy to make an argument for most features being helpful for to those with little experience, given that is our majority user demographic.

Reply:
i agree, it's hard to draw the line of what features should be added, i think that it is hard to grasp all the use cases of the p5.js features, so people can easily propose a new feature, and adding it seems like little work, but it involves a lot of work on the development, or the proposed feature, is a corner case of new features.

something that i am always thinking is that new features, longer documentation, more dependencies add more chances for breaking, adds more barriers to people who want to read the source code, or help out. i will keep on thinking about this too, i don't know a good way to phrase this or explain this :)

Reply:
Great start! But I want us to start expanding and going beyond words such as "'underrepresented’, ‘underserved’, ‘marginalized’ or ‘minority." I know this is a longer conversation about semantics, but in the past few years, there has been a push to not use these words. More soon!

Reply:
@drsantos8791 I think we may need more help here. Do you have suggestions for alternative phrasing? For example in these phrases: 
- `We prioritize the needs of historically marginalized groups over the continued comfort of more privileged groups with p5.js.`
- `Mentoring and supporting learners of p5.js within communities that are historically excluded from and marginalized in creative coding and the digital arts`

Reply:
@lmccart At the moment, and after reading through several articles on this issue, I don't have any further suggestions. Also, I feel these statements best exemplify the mission and objective of what we're trying to do. If I come across suggestions, I will gladly share them with the group. But at the moment, I don't have alternatives. Thank you for checking in.

Reply:
Thank you @drsantos8791! And this is a living document so we can continue to update as we learn more and develop our language and thinking.